validation/src/main/scala/hmda/validation/rules/lar/quality/Q030.scala

```diff
@@ -1,48 +1,20 @@
 package hmda.validation.rules.lar.quality
 
-import hmda.model.fi.lar.{ Geography, LoanApplicationRegister }
-import hmda.model.institution.Institution
-import hmda.validation.context.ValidationContext
+import hmda.model.fi.lar.LoanApplicationRegister
 import hmda.validation.dsl.PredicateCommon._
-import hmda.validation.dsl.PredicateGeo._
 import hmda.validation.dsl.PredicateSyntax._
-import hmda.validation.dsl.{ Failure, Result, Success }
-import hmda.validation.rules.{ EditCheck, IfInstitutionPresentIn }
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
 
-class Q030 private (institution: Institution) extends EditCheck[LoanApplicationRegister] {
+object Q030 extends EditCheck[LoanApplicationRegister] {
   override def name: String = "Q030"
 
   override def apply(lar: LoanApplicationRegister): Result = {
     when(lar.actionTakenType is containedIn(1 to 6)) {
-      val geo: Geography = lar.geography
-      (geo, institution.cra) match {
-        case (Geography("NA", "NA", "NA", "NA"), false) => Success()
-        case (Geography(_, "NA", _, _), _) => Failure()
-        case (Geography(_, _, "NA", _), _) => Failure()
-        case _ =>
-          when(institution.cra is false) {
-            when(geo is smallCounty) {
-              geo.tract is "NA"
-            }
-          } and when(geo.tract is "NA") {
-            geo is smallCounty // and therefore also a valid state/county combination; no need to check again.
-          } and when(geo.msa not "NA") {
-            (geo is validStateCountyMsaCombination) and
-              when(geo.tract not "NA") {
-                geo is validCompleteCombination
-              }
-          } and when(geo.msa is "NA") {
-            when(geo.tract not "NA") {
-              geo is validStateCountyTractCombination
-            }
-          }
-      }
+      (lar.geography.msa not "NA") and
```

Comment:
This only works if the user exactly follows the FIG, using `"NA"` and not `"na"` or `"not applicable"` or `""` or anything. After Juan's #1336 is in, do we need to protect against any of those options?

Reply:
[This line](https://github.com/cfpb/hmda-platform/pull/1336/files#diff-d44a9bbfd5df9409b42d4f4649baa8d2R175) in his PR means that all the values will be strictly "NA".